lib/targets/cuda/target_cuda.cmake

```diff
@@ -103,8 +103,11 @@ if(CMAKE_CUDA_COMPILER_ID MATCHES "NVIDIA" OR CMAKE_CUDA_COMPILER_ID MATCHES "NV
 endif()
 cmake_dependent_option(QUDA_HETEROGENEOUS_ATOMIC "enable heterogeneous atomic support ?" ON
                        "QUDA_HETEROGENEOUS_ATOMIC_SUPPORT" OFF)
+cmake_dependent_option(QUDA_HETEROGENEOUS_ATOMIC_INF_INIT "use infinity as the sentinal for heterogeneous atomic support?" ON
```

Comment:
`sential` -> `signal`?

Reply:
no, it should be `sentinel` which it seems I can't spell

Reply:
a860b1cfe86ed079cdaef3352f867d35b8dcc86f